pkg/database/migrations.go

```diff
@@ -1827,6 +1828,21 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return nil
 			},
 		},
+		{
+			ID: "00076-EnableExtension_hstore",
```

Comment:
can these not be combined?

seems plausible that hstore needs to be committed before it can be used, just curious.

Reply:
I want to make sure the transaction doesn't fail (if it can't find hstore type before it is committed)

Reply:
Yea, it should be separate as-is here.